pkg/bugzilla/bugzilla_test.go

```diff
@@ -0,0 +1,20 @@
+package bugzilla
+
+import "testing"
+
+func TestValidateBZRequestError(t *testing.T) {
+	goodResponse := &bugzillaRequestError{statusCode: 201, bugzillaCode: 201, message: "I am an error"}
+	badResponse := &bugzillaRequestError{statusCode: 301, bugzillaCode: 301, message: "I am an error"}
+	isItAnError := validateBZRequestError(goodResponse)
+	if isItAnError {
+		t.Fatalf("failed to validate whitelisted return codes")
+	}
+	isItAnError = validateBZRequestError(nil)
+	if isItAnError {
+		t.Errorf("failed to validate nil error")
+	}
+	isItAnError = validateBZRequestError(badResponse)
+	if !isItAnError {
+		t.Fatalf("wrong validataion of statusCode %d", badResponse.statusCode)
+	}
+}
```

Comment:
I know it's a bit of an overkill, for such a simple test, but you should have a single test with multiple testCases, like:
https://github.com/openshift/release-controller/blob/d6ec05ffa978dd04b8ba878757557d54694eba5b/pkg/release-controller/release_info_test.go#L8-L40